File: comb-publish-interface/src/main/java/com/comb/publish/service/TeamUserService.java
package com.comb.publish.service;


import java.util.List;
import com.comb.pojo.TbTeamUser;

import entity.PageResult;
/**
 * 服务层接口
 * @author WangQingBang
 *
 */
public interface TeamUserService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbTeamUser> findAll();
	
	
//	/**
//	 * 返回分页列表
//	 * @return
//	 */
//	public PageResult findPage(int pageNum,int pageSize);
//	
	
	/**
	 * 增加
	*/
	public void add(TbTeamUser teamUser);
	
	
	/**
	 * 修改
	 */
	public void update(TbTeamUser teamUser);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbTeamUser findOne(String id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(String[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(TbTeamUser teamUser, int pageNum,int pageSize);
	/**
	 * 更新状态
	 * @param teamId
	 * @param status
	 */
	public void updateStatus(String teamId, String status);
	
	/**
	 * 判断输入密码是否正确
	 * @param id
	 * @param oldPassword
	 * @return
	 */
	public boolean isPassword(String id,String oldPassword);
	
	/**
	 * 修改密码
	 * @param id
	 * @param password
	 */
	public void updataPassword(String id, String password);
}

File: comb-manager-school-web/src/main/webapp/js/service/teamUserService.js
//服务层
app.service('teamUserService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../teamUser/findAll.do');		
	}
//	//分页 
//	this.findPage=function(page,rows){
//		return $http.get('../teamUser/findPage.do?page='+page+'&rows='+rows);
//	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../teamUser/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../teamUser/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../teamUser/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../teamUser/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../teamUser/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
	//更新状态
	this.updateStatus = function(teamId,status){
		return $http.get('../teamUser/updateStatus.do?teamId='+teamId+"&status="+status);
	}
});
